<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mu-image</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>15</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="actionResize"/>
   <addaction name="actionCrop"/>
   <addaction name="actionTransparency"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomDefault"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionHelp"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QDockWidget" name="dockPanels">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockPanelsContent">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedPanels">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="emptyPanel"/>
       <widget class="QWidget" name="scalePanel">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="scaleHeightPxSpinBox"/>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="scaleWidthPcSpinBox"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>px</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="0" colspan="3">
          <widget class="QCheckBox" name="proportionsCheckBox">
           <property name="text">
            <string>Keep proportions</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="scaleHeightPcSpinBox"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>px</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="scaleWidthPxSpinBox"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QPushButton" name="resizeButton">
           <property name="text">
            <string>Resize</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Scale</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="cropPanel">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>px</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="cropYSpinBox"/>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Crop</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QSpinBox" name="cropHeightSpinBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="12" column="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>px</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>px</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="cropWidthSpinBox"/>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="cropXSpinBox"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>px</string>
           </property>
          </widget>
         </item>
         <item row="15" column="0" colspan="3">
          <widget class="QPushButton" name="cropButton">
           <property name="text">
            <string>Crop</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../resources/mu-image.qrc">
     <normaloff>:/resources/load.png</normaloff>:/resources/load.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load an image</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../resources/mu-image.qrc">
     <normaloff>:/resources/quit.png</normaloff>:/resources/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resources/mu-image.qrc">
     <normaloff>:/resources/save.png</normaloff>:/resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="icon">
    <iconset resource="../resources/mu-image.qrc">
     <normaloff>:/resources/resize.png</normaloff>:/resources/resize.png</iconset>
   </property>
   <property name="text">
    <string>Resize</string>
   </property>
   <property name="toolTip">
    <string>Resize</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="icon">
    <iconset resource="../resources/mu-image.qrc">
     <normaloff>:/resources/crop.png</normaloff>:/resources/crop.png</iconset>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
   <property name="toolTip">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionTransparency">
   <property name="icon">
    <iconset resource="../resources/mu-image.qrc">
     <normaloff>:/resources/transparency.png</normaloff>:/resources/transparency.png</iconset>
   </property>
   <property name="text">
    <string>Transparency</string>
   </property>
   <property name="toolTip">
    <string>Transparency</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../resources/mu-image.qrc">
     <normaloff>:/resources/help.png</normaloff>:/resources/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Help</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../resources/mu-image.qrc">
     <normaloff>:/resources/zoom-in.png</normaloff>:/resources/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoomDefault">
   <property name="icon">
    <iconset resource="../resources/mu-image.qrc">
     <normaloff>:/resources/zoom-1.png</normaloff>:/resources/zoom-1.png</iconset>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
   <property name="toolTip">
    <string>Default Zoom Leve</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../resources/mu-image.qrc">
     <normaloff>:/resources/zoom-out.png</normaloff>:/resources/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>cropXSpinBox</tabstop>
  <tabstop>cropYSpinBox</tabstop>
  <tabstop>cropWidthSpinBox</tabstop>
  <tabstop>cropHeightSpinBox</tabstop>
  <tabstop>cropButton</tabstop>
  <tabstop>scaleWidthPcSpinBox</tabstop>
  <tabstop>scaleHeightPxSpinBox</tabstop>
  <tabstop>scaleHeightPcSpinBox</tabstop>
  <tabstop>resizeButton</tabstop>
  <tabstop>proportionsCheckBox</tabstop>
  <tabstop>scaleWidthPxSpinBox</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/mu-image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionResize</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_resize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCrop</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_crop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTransparency</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_transparency()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resizeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>apply_resize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cropButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>apply_crop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomIn</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_zoom_in()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomDefault</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_zoom_default()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomOut</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_zoom_out()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>do_load()</slot>
  <slot>do_save()</slot>
  <slot>do_help()</slot>
  <slot>do_resize()</slot>
  <slot>do_crop()</slot>
  <slot>do_transparency()</slot>
  <slot>apply_resize()</slot>
  <slot>apply_crop()</slot>
  <slot>apply_transparency()</slot>
  <slot>do_zoom_in()</slot>
  <slot>do_zoom_default()</slot>
  <slot>do_zoom_out()</slot>
 </slots>
</ui>
